src/main/java/it/eng/idsa/businesslogic/routes/CamelRouteSender.java

```diff
@@ -322,6 +322,12 @@ public void configure() throws Exception {
                 .process(senderUsageControlProcessor)
                 .process(registerTransactionToCHProcessor)
 				.process(sendResponseToDataAppProcessor);
+			
+			int test = configuration.getWssSelfDescriptionPort();
```

Comment:
Could we name this property somehow different, more self describing?

Reply:
It will be fixed, it's a leftover, and it can be directly called in the next line:

from("jetty://https4://0.0.0.0:" +  configuration.getWssSelfDescriptionPort() + "/internal/sd")